### ISSUE TITLE
Getting message "has UDP ports still in use" when I shut down project

### ISSUE THREAD

**Issue by wdennis** [GNS3/gns3-server#323]
On 1.4.0b3, when I shut down a project that has QEMU machines connected by a link, I always get a "has UDP ports still in use" message after the project closed message, eg:
```
2015-09-28 08:22:42 INFO project.py:380 Project e1caa8ec-06a2-44e3-aca0-7af8bb02272b with path '/home/gns3user/GNS3/projects/e1caa8ec-06a2-44e3-aca0-7af8bb02272b' closed
2015-09-28 08:22:42 WARNING project.py:385 Project e1caa8ec-06a2-44e3-aca0-7af8bb02272b has UDP ports still in use: {10000, 10001}
```
Checking the list of open UDP ports with `ss`, I do not see these ports in the list, so think it may be a spurious message.

Was also happening on 1.4.0b2 as well.

**Comment by grossmj** [comment]
The issue is present in version 1.3.11 as well.

**Comment by grossmj** [comment]
This will have to be merged in 1.4

**Comment by noplay** [comment]
Merged to 1.4
